docs/guides/implementation-guides/how-to-implement-dependency-injection.md

```diff
@@ -7,9 +7,120 @@ import MvvmArchitectureDiagram from '/img/gitbook-import/assets/image (3) (1) (2
 
 # How To Implement Dependency Injection
 
-Content in preparation.
+:::info
+For views you can make a simple [ViewLocator](../../concepts/view-locator) which will automatically resolve the view from the Locator.
+:::
 
 This guide will show you how to use Dependency Injection (DI) with _Avalonia UI_ and the MVVM pattern. 
 
 <img src={MvvmArchitectureDiagram} alt=""/>
 
+Let's assume that your MainViewModel as a dependency on MyService. The contructor for ViewModel should looks like this:
+```csharp
+private readonly MyService _myService;
+
+public MapsViewModel(MyService myService)
+{
+    _myService = myService;
+}
+```
+These are the steps you need to do in order to resolve that dependencu using DI.
```

Comment:
Please add more information. The code you copied from me specifically is for using DI in a multi platform way across platform specific projects where you might have platform specific service implementations. There are simpler ways of doing this if you are only dealing with a single project/shared platform (such as desktops).

Reply:
I was not aware that there is a simplest way for this scenario to be honest. The app I've created using the wizard seems to be setup like a multi-platform one even if it's only desktop. I'm happy to add more info but you'll need to give me more details as I lack experience on this.